src/graph/topo.cc

```diff
@@ -369,7 +369,17 @@ ncclResult_t ncclTopoAddNic(struct ncclXmlNode* xmlNic, struct ncclTopoSystem* s
 
 ncclResult_t ncclTopoAddGpu(struct ncclXmlNode* xmlGpu, struct ncclTopoSystem* system, struct ncclTopoNode* gpu) {
   NCCLCHECK(xmlGetAttrInt(xmlGpu, "sm", &gpu->gpu.cudaCompCap));
-  NCCLCHECK(xmlGetAttrInt(xmlGpu, "gcn", &gpu->gpu.gcn));
+  NCCLCHECK(xmlGetAttr(xmlGpu, "gcn", &gpu->gpu.gcn));
+  if (strcmp(gpu->gpu.gcn, "906") == 0) {
+    //NCCLCHECK(xmlSetAttr(gpu->gpu, "gcn", "gfx906"));
+    gpu->gpu.gcn = "gfx906";
+  } else if (strcmp(gpu->gpu.gcn, "908") == 0) {
+    //NCCLCHECK(xmlSetAttr(gpu->gpu, "gcn", "gfx908"));
+    gpu->gpu.gcn = "gfx908";
+  } else if (strcmp(gpu->gpu.gcn, "910") == 0) {
+    //NCCLCHECK(xmlSetAttr(gpu->gpu, "gcn", "gfx90a"));
+    gpu->gpu.gcn = "gfx90a";
+  }
```

Comment:
Is there any reason to keep the extra NCCLCHECK comments?

Reply:
hahaha, nope, just forgot to delete em.  Thanks!